### DISCUSSION TITLE
Change de pitch

### DISCUSSION THREAD

**Discussion by DiegoBari** [PaulWoitaschek/Voice#2190] [Ideas]
I would like to see the pitch change function added, so that changing the playback speed does not change the pitch of the voice so much. I saw it implemented in the [NewPipe](https://newpipe.net/) player. Thanks

**Comment by ghost** [comment]
Changing playback speed does not change the pitch. Must be a bug specific to your device.

**Reply by ghost** [reply]
Change pitch would be great option for my tinnitus and peaceful in ears, especially when is male voice in books so change pitch will rescue for me. Thank you
